docs/build/examples/simple-pinning-service/step-1-powergate-setup.md

```diff
@@ -18,16 +18,16 @@ To follow this tutorial you will need:
 
 To set up Powergate:
 
-1. Download the `powergate-docker-<version>.zip` from the appropriate release, in this case, [version v0.0.1-beta.10](https://github.com/textileio/powergate/releases/download/v0.0.1-beta.10/powergate-docker-v0.0.1-beta.10.zip).
+1. Download the `powergate-docker-<version>.zip` from the appropriate release, in this case, [version v0.4.1](https://github.com/textileio/powergate/releases/download/v0.4.1/powergate-docker-v0.4.1.zip).
```

Comment:
Do you think we should just point the readers to the powergate releases and remove the version name from the docs altogether? Otherwise, we would end up updating the docs again and again.